Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -683,6 +683,24 @@ final class UIViewExtensionsTests: XCTestCase { // swiftlint:disable:this type_b
         view.removeBlur()
         XCTAssertFalse(view.subviews.first is UIVisualEffectView)
     }
+    
+   
+    func testAddViewSeparator() throws {
+        let view = UIView()
+        let expectedColor: UIColor = .red
+        let expectedHeight: CGFloat = 2.0
+
+        view.addViewSeparator(color: expectedColor, height: expectedHeight)
+
+        let separator = try XCTUnwrap(view.subviews.first, "Separator not found")
+        
+        XCTAssertEqual(view.subviews.count, 1)
+        XCTAssertEqual(separator.backgroundColor, expectedColor)
+        XCTAssertEqual(separator.frame.height, expectedHeight, accuracy: 0.1)
+    }
+
+    
+    
```

Comment:
There are excessive empty spaces here. Do you have `swiftformat` and `swiftlint` installed? If not, please install them to make sure that the code meets the project standards.